### ISSUE TITLE
Gold graph does not match the in-game networth

### ISSUE THREAD

**Issue by zachkont** [odota/web#2548]
**Current Behavior**
In-game hero networth and opendota graph numbers don't match

**Expected behavior/code**
I would have expected the gold graphs and in-game networth numbers to be the same

**Console Output**
N/A - This is probably not a web bug but the `report a bug` button linked this repo so I just went ahead, sorry if it's not the right place

**Environment**
- Browser Version: [e.g. Chrome 64.0, Firefox 57, Edge 17]
- OS: [e.g. OSX 10.13.4, Windows 10 or Linux Distribution]

**Possible Solution**
The numbers match the in-game networth number or more realistically the gold calculation formula and possible discrepancy reason is explained in the site (maybe a tooltip or something)

**Additional context/Screenshots**
A clear and concise description of the behavior.
![image](https://user-images.githubusercontent.com/17274058/80347427-f4a10a80-8874-11ea-9dd4-ef3531d0f11a.png)
![image](https://user-images.githubusercontent.com/17274058/80347470-fff43600-8874-11ea-9f94-43c73029752e.png)

https://www.opendota.com/matches/5380142389/graphs

**Comment by howardchung** [comment]
This is because the stat we're extracting from the replay is 'gold earned' not 'net worth'. I believe net worth is in a separate field that we could display separately if we wanted

**Comment by zachkont** [comment]
Yeah I imagined it was something like that but I can't understand how the MK can have less gold earned than he has networth. Does it not count the starting gold either?

Also it would be very cool to show both graphs imo
